### PR TITLE
Added favicon

### PR DESCRIPTION
**Pull request by Ashutosh00710**
Added favicon for the community site

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <title>Eddie Jaoude Community</title>
 
-    <link rel="icon" type="image/png" href="/favicon.png" />
+    <link rel="icon" type="image/ico" href="./favicon.ico" />
```
Comment (eddiejaoude) [inline public/index.html:9]: This file does not exist?

Comment (Ashutosh00710) [reply public/index.html:9]: image

Yes sir it is right there. ☝🏻

Comment (BOLT04) [reply public/index.html:9]: yes, this file is also added on this PR in the public folder 😃

**Review by BOLT04 — APPROVED** [review]
LGTM 👍. This is how it looks:
imagem

File: public/index.html (hunk 1 above)
Comment (eddiejaoude) [reply public/index.html:9]: Oops sorry I missed it 🤦‍♂️

**Comment by eddiejaoude** [comment]
Thank you both 👍

**Pull request merged**